content/enterprise/admin/infrastructure/demo-to-disk-migration.mdx

```diff
@@ -0,0 +1,104 @@
+---
+page_title: Demo to Mounted Disk Migration - Infrastructure Administration - Terraform Enterprise
```

Comment:
Can you add page metadata description here, too? 150 characters that could show up as the google search snippet!  There's an example of one in the markdown of this page: https://www.terraform.io/intro

Reply:
Added!